src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -289,13 +290,16 @@ private void MethodChecks(MethodDeclarationSyntax syntax, Binder withTypeParamsB
 
             if (IsPartial)
             {
-                // check that there are no out parameters in a partial
-                foreach (var p in this.Parameters)
+                if (MessageID.IDS_FeatureExtendedPartialMethods.GetFeatureAvailabilityDiagnosticInfo(DeclaringCompilation) is { } info)
                 {
-                    if (p.RefKind == RefKind.Out)
+                    // check that there are no out parameters in a partial
+                    foreach (var p in this.Parameters)
                     {
-                        diagnostics.Add(ErrorCode.ERR_PartialMethodCannotHaveOutParameters, location);
```

Comment:
>ERR_PartialMethodCannotHaveOutParameters [](start = 50, length = 40)

If we are no longer reporting this error, please consider commenting out the `ErrorCode` field and removing the resource string.

Reply:
Done